wp-content/themes/core/components/link/link.php

```diff
@@ -15,7 +15,16 @@
 ); ?>
 
 <a <?php echo $c->get_classes(); ?> <?php echo $c->get_attrs(); ?>>
+
+	<?php if ( $c->icon_has_classes() && $c->is_icon_before() ) : ?>
+			<i  <?php echo $c->get_icon_classes(); ?> aria-hidden="true"></i>
+	<?php endif; ?>
+
 	<?php echo $c->get_content(); ?>
+
+	<?php if ( $c->icon_has_classes() && ! $c->is_icon_before() ) : ?>
+			<i  <?php echo $c->get_icon_classes(); ?> aria-hidden="true"></i>
+	<?php endif; ?>
```

Comment:
not sure how i feel about using an <i> tag. Might want to discuss this as a group but I'd vote a span

Reply:
Reviewing with Jamie and Dave, they both suggested an `<i>`. AFAIK the font icon docs also recommend this. But open if a span would be better.

Reply:
Cool I'll change it.

Reply:
case in point was it changed my comment to italic haha